README.md

```diff
@@ -8,11 +8,11 @@ We thought the best way to do this is have you **pick two** of the following way
 
 Either:
 
-* *Pick Two* level-appropriate hiring exercises ("Level 2" exercises are prefixed with `2-`)
+* *Pick Two* hiring exercises from the list given to you
```

Comment:
maybe tack on to the end of this "by your interview coordinator" for extra clarity?